### ISSUE TITLE
Upgrade to Node 10

### ISSUE THREAD

**Issue by chalin** [dart-lang/site-webdev#1775]

**Comment by chalin** [comment]
I was having issues running commands:

```console
> gulp e2e --filter=toh-5
File: /Users/chalin/dev/log/wd-run-log-180809-3.txt
DRUN $Id: drun,v 1.1 2006/06/03 05:17:28 chalin Exp 92478
Invoked as: gulp e2e --filter=toh-5
PWD: /Users/chalin/git/wd
Start at:  Thu 9 Aug 2018 17:00:13 EDT
branch: * chalin-node-10-0809
------------------------------------------------------------------
gulp e2e --filter=toh-5[92486]: ../src/node_contextify.cc:637:static void node::contextify::ContextifyScript::New(const FunctionCallbackInfo<v8::Value> &): Assertion `args[1]->IsString()' failed.
 1: 0x100038d1f node::Abort() [/Users/chalin/.nvm/versions/node/v10.8.0/bin/node]
 2: 0x100037e19 node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, char const*, int, v8::Local<v8::Value>*, node::async_context) [/Users/chalin/.nvm/versions/node/v10.8.0/bin/node]
 3: 0x100060837 node::contextify::ContextifyScript::New(v8::FunctionCallbackInfo<v8::Value> const&) [/Users/chalin/.nvm/versions/node/v10.8.0/bin/node]
 4: 0x10023bf8f v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo*) [/Users/chalin/.nvm/versions/node/v10.8.0/bin/node]
 5: 0x10023b128 v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<true>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) [/Users/chalin/.nvm/versions/node/v10.8.0/bin/node]
 6: 0x10023aaf7 v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) [/Users/chalin/.nvm/versions/node/v10.8.0/bin/node]
 7: 0xfaa192041bd 
------------------------------------------------------------------
```
This comment helped: https://github.com/nodejs/node/issues/20285#issuecomment-387612673

```console
> rm -Rf node_modules/ package-lock.json 
> npm cache clean --force
> ./scripts/before-install.sh # etc
```

**Comment by chalin** [comment]
Closed by #1774
